turbo-repo/apps/app/src/lang/es.json

```
@@ -2663,6 +2663,142 @@
+      "viewer_prev": "Anterior",
+      "viewer_next": "Siguiente",
+      "viewer_close": "Cerrar visor",
+      "viewer_click_rename": "Clic para renombrar",
+      "viewer_link_copied": "Enlace copiado al portapapeles",
+      "viewer_loading_doc": "Cargando documento...",
+      "viewer_load_error": "No se pudo cargar el documento",
+      "viewer_change_category": "Cambiar tipo de documento",
+      "category_change_loading": "Actualizando tipo de documento...",
+      "category_change_success": "Tipo de documento actualizado",
+      "category_change_error": "Error al actualizar el tipo de documento",
+      "status_pending": "Pendiente",
```

Biome (2.4.15)

[error] The key status_pending was already declared.

(lint/suspicious/noDuplicateObjectKeys)